### PR TITLE
CORE-12019 Add a "stopAndCleanCorda" gradle task that will Stop Corda5 if running and do some cleanup

### PR DESCRIPTION
**Pull request by egabrovski-r3**
moved from https://github.com/corda/CSDE-cordapp-template-kotlin/pull/59

### CODE REVIEW

File: buildSrc/src/main/groovy/csde.gradle [hunk 1]
```diff
@@ -148,6 +148,18 @@ tasks.register("stopCorda") {
     }
 }
 
+tasks.register("stopAndCleanCorda") {
+    group = cordaGroup
+    dependsOn('clean')
+    doLast {
+        try {
+            cordaLifeCycle.stopCorda()
+        } catch (Exception ignored) {
```
Comment (yarivus) [inline buildSrc/src/main/groovy/csde.gradle]: I think it's OK to ignore ... but at least we should log it stating it's ignored.
Hiding exceptions completely may confuse later users

**Review by yarivus — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/groovy/csde.gradle (hunk 1 above)
Comment (egabrovski-r3) [reply buildSrc/src/main/groovy/csde.gradle]: changed to print the exception

File: buildSrc/src/main/groovy/csde.gradle [hunk 2]
```diff
@@ -154,7 +154,8 @@ tasks.register("stopAndCleanCorda") {
     doLast {
         try {
             cordaLifeCycle.stopCorda()
-        } catch (Exception ignored) {
+        } catch (Exception ignore) {
+            println(ignore.toString())
```
Comment (yarivus) [inline buildSrc/src/main/groovy/csde.gradle]: I think you need to state that you're ignoring to your fellow developer who will debug this in the future:
```suggestion
            println("ignoring exception: ${ignore.toString()}")
```

**Review by yarivus — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/groovy/csde.gradle (hunk 2 above)
Comment (egabrovski-r3) [reply buildSrc/src/main/groovy/csde.gradle]: done

**Review by yarivus — APPROVED** [review]
LGTM

**Comment by egabrovski-r3** [comment]
qa-regpack-csde tests are Green

**Pull request merged**